ontology/observable/observable.ttl

```diff
@@ -1610,16 +1610,16 @@ observable:DeviceFacet
 	rdfs:comment "A device facet is a grouping of characteristics unique to a piece of equipment or a mechanism designed to serve a special purpose or perform a special function. [based on https://www.merriam-webster.com/dictionary/device]"@en ;
 	sh:property
 		[
-			sh:datatype xsd:string ;
+			sh:class core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the observable namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.